client/components/foldable-card/index.jsx

```diff
@@ -41,7 +41,7 @@ var FoldableCard = React.createClass( {
 			onClose: noop,
 			cardKey: '',
 			icon: 'chevron-down',
-			isExpanded: false
+			isExpanded: false,
```

Comment:
I thought it was good practice to end all lines in objects with a comma to avoid double-line diffs when a single line gets added (clarity) and making all the lines uniform.